### ISSUE TITLE
README file for the training code

### ISSUE THREAD

**Issue by HarryHsing** [ByteDance-Seed/m3-agent#1]
Thank you so much for the great work! 

Will you be updating the README file with more details about the training process later?

**Comment by HarryHsing** [comment]
> Thank you for your attention to our work. The training data consists of internal videos and cannot be open-sourced. However, the data construction method is similar to that of M3-Bench, which has been described in detail in the paper. For specific training details, please refer to the "training" and "appendix" sections of the paper.

Thanks a lot for the prompt response. Will you release the training code?
